### DISCUSSION TITLE
why there is no official document explaining the namespace and group feature for translation?

### DISCUSSION THREAD

**Discussion by videni** [laravel/framework#42410] [Q&A]
This is quite strange , the official doc [Laravel Localization](https://laravel.com/docs/9.x/localization) doesn't mention translation namespace and group at all, why is that?

**Comment by madalinignisca** [comment]
Can you explain what do you mean by namespace and group related to localization?

**Comment by LasseRafn** [comment]
Also very curious now 😄 sounds interesting.

**Comment by madalinignisca** [comment]
@videni, translations are simply returned arrays and allow you to have any custom ones overriding packages or whatever. There is no namespace, identical to how config/* ain't having. This php files simply return multi-level arrays with strings...
